transform/app/settings.py

```diff
@@ -72,6 +79,22 @@ class GlobalSettings(BaseSettings):
     GUIDELINE: str = "interoperability guideline"
     TRAINING: str = "training"
 
+    # Relations properties
+    ORGANIZATION: str = "organization"
```

Comment:
This is correct. However, in the code where I iterate through directories in the 'directory' column, I save the name of the directory. Therefore, when I want to map it, I must use 'organization' instead of 'organisation', which is the name of our data directory.